src/helpers/price-labels.ts

```diff
@@ -1,4 +1,5 @@
-import { parseISODuration, type Period, PeriodUnit } from "./duration-helper";
+import { parseISODuration } from "./duration-helper";
+import { Translator } from "../ui/localization/translator";
```

Comment:
```suggestion
import type { Translator } from "../ui/localization/translator";
```

Reply:
Please check the formatting PR here:
https://github.com/RevenueCat/purchases-js/pull/251